CMakeLists.txt

```diff
@@ -92,6 +92,16 @@ endif()
 
 include(CTest)
 
+if(MSVC)
+	# This option is only available when building with MSVC. By default, highs
+	# is build using the cdecl calling convention, which is useful if you're
+	# writing C. However, the CLR and Win32 API both expect stdcall.
```

Comment:
I guess you are talking about P/Invoke. I always use cdecl and it works fine with explicite calling convention [DllImport(CallingConvention = CallingConvention.Cdecl)].
Therefore this description imho way to simplified.
Are you sure that "if(MSVC)" is sufficient? What's about mingw, clang and the intel compiler?